app/controllers/application_controller.rb

```diff
@@ -1892,7 +1892,8 @@ def controller_for_common_methods
       "embedded_configuration_script_payload"
     when "workflow_repository"
       "ansible_repository"
-    # TODO: Update manageiq/db/fixtures/miq_product_features.yml by replacing ansible_credential_tag with embedded_automation_manager_credential_tag
+    # DONE: Update manageiq/db/fixtures/miq_product_features.yml by replacing ansible_credential_tag with embedded_automation_manager_credential_tag
+    # TODO: This is now done, can we fix this case statement?
```

Comment:
@GilbertCherrie not sure if we can fix this case statement now that we rename this.  It was added in 1cc6e74614fe063e5f3b50495004fd9c95f7a829 and treats workflows like ansible repositories..